fuels-abigen-macro/tests/test_projects/call_empty_return/Forc.toml

```
@@ -5,5 +5,5 @@ name = "contract_test"
 entry = "main.sw"
 
 [dependencies]
-std = { path = "../lib-std" }
-core = { path = "../lib-core" }
+std = { git = "https://github.com/FuelLabs/sway-lib-std"}
+core = { git = "https://github.com/FuelLabs/sway-lib-core/" }
```

For all these, could you
1. Sort alphabetically
2. Remove `/` at the end of `core`'s path
3. Add a space for `std`
```suggestion
core = { git = "https://github.com/FuelLabs/sway-lib-core" }
std = { git = "https://github.com/FuelLabs/sway-lib-std" }
```